.changeset/grumpy-laws-rule.md

```diff
@@ -0,0 +1,5 @@
+---
+'@compiled/babel-plugin': patch
```

Comment:
I don't think we need to bump babel plugin for this, since the changes are just tests

Reply:
Looks like changeset changed their recommendations, before it mentioned to include a changeset for everything even if it's not released